src/OpenTelemetry.Exporter.Zipkin/ZipkinExporterOptions.cs

```diff
@@ -34,7 +34,7 @@ public sealed class ZipkinExporterOptions
         /// Gets or sets Zipkin endpoint address. See https://zipkin.io/zipkin-api/#/default/post_spans.
         /// Typically https://zipkin-server-name:9411/api/v2/spans.
         /// </summary>
-        public Uri Endpoint { get; set; } = new Uri("http://localhost:9411/api/v2/spans");
+        public Uri Endpoint { get; set; } = new Uri(Environment.GetEnvironmentVariable("OTEL_EXPORTER_ZIPKIN_ENDPOINT") ?? "http://localhost:9411/api/v2/spans");
```

Comment:
We could add a unit test to confirm this behavior.

Reply:
@utpilla - Thanks for the review. Added test now - please review.

Reply:
LGTM.

Reply:
If this failed (throws exception such as Invalid URI), would it block the app? (e.g. type initialization exception or bad app domain)

Reply:
Yes - this behavior is same as setting uri in code. if invalid, it will break the app.